.github/ISSUE_TEMPLATE/new-ontology-request.md

```diff
@@ -49,3 +49,28 @@ GitHub username:
 
 
 ## Additional comments or remarks
+
+## Metadata
+
+Please fill in the following metadata record which will be used by the OBO Foundry website.
+
+```
+id: yourfourletterid
+title: The Title Of Your Ontology
+contact:
+  email: email@email.fr
```

Comment:
the .fr might be confusing to some. how about
youremail@domain.com

Reply:
I changed it to tdl :D to be very general.